library/std/src/thread/current.rs

```diff
@@ -133,12 +130,32 @@ pub(super) fn set_current(thread: Thread) -> Result<(), Thread> {
     Ok(())
 }
 
-/// Gets the id of the thread that invokes it.
+/// Gets the unique identifier of the thread which invokes it.
+///
+/// Calling this function may be more efficient than accessing the current
+/// thread id through the current thread handle. i.e. `thread::current().id()`.
 ///
 /// This function will always succeed, will always return the same value for
 /// one thread and is guaranteed not to call the global allocator.
+///
+/// # Examples
+///
+/// ```
+/// #![feature(current_thread_id)]
+///
+/// use std::thread;
+///
+/// let other_thread = thread::spawn(|| {
+///     thread::current_id()
+/// });
+///
+/// let other_thread_id = other_thread.join().unwrap();
+/// assert!(thread::current_id() != other_thread_id);
```

Comment:
```suggestion
/// assert_ne!(thread::current_id(), other_thread_id);
```

Reply:
Okay now I'm a little baffled, I did in fact change this however the current version in the github UI is still showing the old? But locally it shows correctly, and the other change to the issue number is correct. :/